layouts/index.html

```diff
@@ -25,6 +25,9 @@
         </div>
         <div id="features" class="container content">
             <div class="row">
+                <div class="col-md-12">
+                    <h2 class="super-heading">Features</h2>
```

Comment:
This actually kind of breaks the design a bit, IMHO. For the seeing, it is obvious that this section lists features, with the four large icons describing the main features.  I suggest this heading is also hidden.